pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -228,7 +228,6 @@ buildStdenv.mkDerivation ({
     })
   ]
   ++ lib.optional (lib.versionAtLeast version "86") ./env_var_for_system_dir-ff86.patch
-  ++ lib.optional (lib.versionAtLeast version "90" && lib.versionOlder version "95") ./no-buildconfig-ffx90.patch
   ++ lib.optional (lib.versionAtLeast version "96") ./no-buildconfig-ffx96.patch
```

Comment:
some of these can be made unconditional now, but we can do that in a separate PR